WalletWasabi.Tests/UnitTests/KeyManagementTests.cs

```diff
@@ -156,7 +157,7 @@ public void CanGenerateKeys()
 			for (int i = 0; i < 1000; i++)
 			{
 				var isInternal = random.Next(2) == 0;
-				var label = RandomString.Generate(21);
+				var label = (new WalletWasabi.Crypto.Randomness.UnsecureRandom() as IWasabiRandom).GetString(21, Characters.AlphaNumeric);
```

Comment:
This is much uglier and harder to use.

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/3903/commits/c2e998a7356a9f6a8d8fa7f7ea9efaa0b445be60